code/public/index.html

```diff
@@ -1,34 +1,27 @@
 <!DOCTYPE html>
 <html lang="en">
+	<head>
```

Comment:
Looks like you have tabs instead of spaces in the HTML file? Take a look at your VSC settings to make sure tabs are interpreted as 2 spaces

Reply:
Same goes for the other files - it makes it hard to read with tabs

Reply:
Thank you for the input, have changed vs code settings. 😃